CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased
+## [Unreleased]
+### Fixed
+- [JsonSchema] [OpenApi] [GH#595](https://github.com/janephp/janephp/pull/595) Fix "oneOf" with nullable
```

Comment:
Put only JsonSchema here, OpenAPI is also impacted but it's the modification are in JsonSchema not OpenAPI

Reply:
Done